src/main/java/com/datalogics/pdf/samples/manipulation/RemoveInteractivity.java

```diff
@@ -30,45 +30,45 @@
 import java.util.EnumSet;
 
 /**
- * This sample demonstrates how to use the flattening functions to flatten the interactive elements of a PDF. The
- * flattening functionality currently handles forms and annotations. All flattening is done through the FormFlattener's
- * flattenDocument() method, and specific element types can be flattened or ignored by specifying them in the
- * setAnnotationsToBeProcessed() method of the APContext passed in. Note that PDFJT does not currently flatten
+ * This sample demonstrates how to use the flattening functions to remove the interactive elements of a PDF. This
+ * functionality currently handles forms and annotations. All flattening is done through the FormFlattener's
+ * flattenDocument() method, and specific element types can be made non-interactive or ignored by specifying them in the
+ * setAnnotationsToBeProcessed() method of the APContext passed in. Note that PDFJT does not currently process
  * transparency.
  *
  * <p>
- * Below is a list of annotation types, and whether they get flattened or not.
+ * Below is a list of annotation types, and whether they will have their interactivity processed or not.
  * <ul>
- * <li>Text - flattened</li>
- * <li>Link - flattened</li>
- * <li>Line - flattened</li>
- * <li>Square - flattened</li>
- * <li>Circle - flattened</li>
- * <li>Polygon - flattened</li>
- * <li>PolyLine - flattened</li>
- * <li>Highlight - flattened</li>
- * <li>Underline - flattened</li>
- * <li>Squiggly - flattened</li>
- * <li>Ink - flattened</li>
- * <li>StrikeOut - flattened</li>
- * <li>Stamp - flattened</li>
- * <li>Popup - not flattened</li>
- * <li>FileAttachment - not flattened</li>
- * <li>Sound - flattened</li>
- * <li>Movie - flattened</li>
- * <li>3D - flattened</li>
+ * <li>Text - processed</li>
+ * <li>Link - processed</li>
+ * <li>Line - processed</li>
+ * <li>Square - processed</li>
+ * <li>Circle - processed</li>
+ * <li>Polygon - processed</li>
+ * <li>PolyLine - processed</li>
+ * <li>Highlight - processed</li>
+ * <li>Underline - processed</li>
+ * <li>Squiggly - processed</li>
+ * <li>Ink - processed</li>
+ * <li>StrikeOut - processed</li>
+ * <li>Stamp - processed</li>
+ * <li>Popup - not processed</li>
+ * <li>FileAttachment - not processed</li>
+ * <li>Sound - processed</li>
+ * <li>Movie - processed</li>
+ * <li>3D - processed</li>
  * </ul>
  * </p>
  */
-public final class FlattenPdf {
+public final class RemoveInteractivity {
 
-    public static final String OUTPUT_FLATTENED_PDF_PATH = "Flattened.pdf";
+    public static final String OUTPUT_PDF_PATH = "Flattened.pdf";
```

Comment:
I wonder if we shouldn't also rename the output file itself, since we're (mostly) banishing the word 'flatten' from this sample. However, I'm not sure what the new name should be; "RemovedInteractivity.pdf" is kind of a mouthful.